src/components/Form/TextInputAdornment.js

```diff
@@ -6,14 +6,14 @@ import styles from './TextInputAdornment.mod.scss'
 
 const cx = className.bind(styles)
 
-const TextInputAdornment = ({ children, position }) => {
+const TextInputAdornment = ({ children, position, style }) => {
   const textInputAdornmentClasses = cx('adornment', {
     end: position === 'end',
     start: position === 'start',
   })
 
   return (
-    <p className={textInputAdornmentClasses}>
+    <p className={textInputAdornmentClasses} style={style}>
```

Comment:
Can we avoid this here? Can you maybe instead use a prop to set a certain type of style?

Reply:
Actually this is not used anywhere else yet, so feel free to set the `padding: 0` from the MarketDetail component that you used in the TextInputAdornments styles and remove the `style` prop